spec/features/user_cart_spec.rb

```diff
@@ -3,8 +3,9 @@
 RSpec.describe 'as visitor', type: :feature do
   describe 'as visitor' do
     it 'lets me add an item to my cart' do
-      item = Item.create(name: 'pot', description:'small pot for plants', quantity: 30, price: 2.49, thumbnail: 'thumbnail.jpeg')
-      visit item_path(item)
+      user = User.create(username: 'bob', street: "1234", city: "bob", state: "bobby", zip_code: 12345, email: "12345@54321", password: "password", role: 0, enabled: 0)
+      item = Item.create!(name: 'pot', description:'small pot for plants', quantity: 30, price: 2.49, thumbnail: 'thumbnail.jpeg', user: user)
```

Comment:
small point, but for people reading the test, it might be better to call the user variable `merchant` instead of `user` to avoid confusion?